src/main/scala/mesosphere/marathon/upgrade/ScalingCalculator.scala

```diff
@@ -0,0 +1,19 @@
+package mesosphere.marathon.upgrade
+
+import mesosphere.marathon.Protos.MarathonTask
+
+object ScalingCalculator {
+
+  def scalingProposition(runningTasks: Set[MarathonTask],
```

Comment:
You could move this to the companion object of ScalingProposition, maybe even call it `apply`.